tt_metal/impl/dispatch/command_queue_interface.hpp

```diff
@@ -14,8 +14,9 @@
 using namespace tt::tt_metal;
 
 // Starting L1 address of commands
-inline uint32_t get_command_start_l1_address() {
-    return L1_UNRESERVED_BASE;
+inline uint32_t get_command_start_l1_address(bool use_eth_l1) {
```

Comment:
this may have been added back by accident. We don't need the `use_eth_l1` toggle

Reply:
it should be `return use_eth_l1 ? ERISC_L1_UNRESERVED_BASE : L1_UNRESERVED_BASE;`

Reply:
That would be this `get_eth_command_start_l1_address` if you're writing to a eth core with user kernels. But that's `eth_l1_mem::address_map:: ERISC_L1_UNRESERVED_BASE` which is different.. I think it's pretty confusing, do we think `ERISC_L1_UNRESERVED_BASE` will be different than `L1_UNRESERVED_BASE`

Reply:
yes it can be different in future.